<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindTool</class>
 <widget class="QWidget" name="FindTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="4">
     <widget class="QCheckBox" name="findCaseCheckBox">
      <property name="text">
       <string>Case sensitive</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="findPrevArrow">
      <property name="text">
       <string/>
      </property>
      <property name="arrowType">
       <enum>Qt::LeftArrow</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="findNextArrow">
      <property name="text">
       <string/>
      </property>
      <property name="arrowType">
       <enum>Qt::RightArrow</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="replaceLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="findLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Find : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="findLineEdit">
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="replaceLabel">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Replace : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="findWholeWordCheckBox">
      <property name="text">
       <string>Whole words</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="replacePushButton">
      <property name="text">
       <string>Replace</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="replaceAllPushButton">
      <property name="text">
       <string>Replace All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
